driver/razerchromacommon.h

```diff
@@ -17,37 +17,51 @@ struct razer_report razer_chroma_standard_get_firmware_version(void);
 /*
  * Standard LED Functions
  */
-struct razer_report razer_chroma_standard_set_led_state(unsigned char variable_storage, unsigned char led_id, unsigned char led_state);
+struct razer_report razer_chroma_standard_set_led_state(unsigned char variable_storage, unsigned char led_id,
+        unsigned char led_state);
```

Comment:
Please don't reformat existing files without any code changes, so please revert those changes in this file and in several other files.

Reply:
OK, But according [Linux kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html) and practice - the line's width 80 is a really better.